install_config/aggregate_logging.adoc

```diff
@@ -355,7 +344,7 @@ $ oc logs -f <pod_name>
 ----
 
 ifdef::openshift-enterprise[]
-. As a cluster administrator, deploy a template that is created by the deployer:
+. As a cluster administrator, deploy the template that the deployer created:
```

Comment:
OK. Changed to:

"As a cluster administrator, deploy the `logging-support-template` template
that the deployer created:"